cmd/vendor_pull.go

```diff
@@ -15,9 +15,6 @@ var vendorPullCmd = &cobra.Command{
 	Long:               `This command executes 'atmos vendor pull' CLI commands`,
 	FParseErrWhitelist: struct{ UnknownFlags bool }{UnknownFlags: false},
 	Run: func(cmd *cobra.Command, args []string) {
-		// Check Atmos configuration
-		checkAtmosConfig()
```

Comment:
Shouldn't we still check the config if stacks are passed? Not sure if removing all config validation is the right fix.

Reply:
For example we should check vending options in the config.